python-sdk/dev/Dockerfile

```diff
@@ -13,6 +13,7 @@ COPY ../pyproject.toml  ${AIRFLOW_HOME}/astro_sdk/
 # The following file are needed because version they are referenced from pyproject.toml
 COPY ../README.md  ${AIRFLOW_HOME}/astro_sdk/
 COPY ../src/astro/__init__.py  ${AIRFLOW_HOME}/astro_sdk/src/astro/__init__.py
+COPY ../src/uto/__init__.py  ${AIRFLOW_HOME}/astro_sdk/src/uto/__init__.py
```

Comment:
@sunank200 what do you think if we used the name `transfer` as opposed to `uto`? From a user's perspective, it may be more intuitive. Acronyms can be confusing!

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1267/commits/e98b4171d221f5b371a67ab945837fa709eb5737)